api/SpeechGrammar.json

```diff
@@ -0,0 +1,64 @@
+{
+  "api": {
+    "SpeechGrammar": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/API/SpeechGrammar",
```

Comment:
This should be the unlocalized docs (i.e. `"https://developer.mozilla.org/docs/Web/API/SpeechGrammar"` 😄